XmlFormatter/src/Windows/MainForm.cs

```diff
@@ -191,104 +190,155 @@ private void MainForm_DragDrop(object sender, DragEventArgs e)
         /// <param name="e">The event arguments</param>
         private void MI_About_Click(object sender, EventArgs e)
         {
-            Version version = GetApplicationVersion();
-            MessageBox.Show(GetStringVersion(version), "Version", MessageBoxButtons.OK, MessageBoxIcon.Information);
+            VersionManager manager = new VersionManager();
+            Version version = manager.GetApplicationVersion();
+            MessageBox.Show(manager.GetStringVersion(version), "Version", MessageBoxButtons.OK, MessageBoxIcon.Information);
         }
 
         /// <summary>
-        /// This method will return you the current version of the application
+        /// This method will check if there is newer version for download
         /// </summary>
-        /// <returns>The current version of the application</returns>
-        private Version GetApplicationVersion()
+        /// <param name="sender">The control triggering the event</param>
+        /// <param name="e">The event data</param>
+        private async void MI_CheckForUpdate_Click(object sender, EventArgs e)
         {
-            Assembly assembly = Assembly.GetExecutingAssembly();
-            string versionString = "0.0.0";
-            using (Stream stream = assembly.GetManifestResourceStream("XmlFormatter.Version.txt"))
+            VersionManager manager = new VersionManager();
+            manager.Error += Manager_Error;
+            VersionCompare versionCompare = await manager.GitHubVersionIsNewer();
+            string text = "Your version is up to date";
+            MessageBoxButtons buttons = MessageBoxButtons.OK;
+            if (versionCompare.GitHubIsNewer)
+            {;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=4995221)